Sources/Equatable/MomModel+Equatable.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  MomModel+Equatable.swift
+//  MomXML
+//
+//  Created by anass talii on 12/06/2017.
+//  Copyright © 2017 phimage. All rights reserved.
+//
+
+import Foundation
+
+extension MomModel: Equatable {
+
+    public static func == (lhs: MomModel, rhs: MomModel) -> Bool {
+        if lhs.entities.count != rhs.entities.count {
+            return false
+        }
+        if lhs.elements.count != rhs.elements.count {
+            return false
+        }
+        for entry in lhs.entities {
+            for entry2 in rhs.entities where entry2 != entry {
+                return false
```

Comment:
I thinks that entities comparaison will always return false here
same for elements. See MomUserInfo for complex implementation
I will fix later with a more beautiful implementation that I do in MomUserInfo